volestipy/volestipy/src/bindings.cpp

```diff
@@ -276,7 +284,76 @@ void HPolytopeCPP::rounding(char* rounding_method, double* new_A, double* new_b,
 }
 //////////         End of "rounding()"          //////////
 
+double HPolytopeCPP::rounding_svd_step(double* new_A, double* new_b,
+                                     double* T_matrix, double* shift,
+                                     double* inner_point, double radius) {
+
+   RNGType rng(HP.dimension());
+   HP.normalize();
+      
+   // if yes, then read the inner point provided by the user and the radius
+   int d = HP.dimension();
+   VT inner_vec(d);
+   
+   for (int i = 0; i < d; i++){
+      inner_vec(i) = inner_point[i];
+   }
+   
+   Point inner_point2(inner_vec);   
+   CheBall = std::pair<Point, NT>(inner_point2, radius);   
+      
+   HP.set_InnerBall(CheBall);   
+   unsigned int walk_length = 2;
+   svd_rounding_single_step<AcceleratedBilliardWalk, MT, VT>(HP, CheBall, walk_length,
+                                                             svd_parameters, rng);
+
+   if (svd_parameters.fail) {
+      std::cout<<"method failed"<<std::endl;
```

Comment:
fail but continues , is this safe?

Reply:
Changed in terms of returning value implying failure